cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -26,19 +26,32 @@
 from cirq_google.experimental import ops as experimental_ops
 
 
-SYC_GATE_FAMILY = cirq.GateFamily(ops.SYC)
-SQRT_ISWAP_GATE_FAMILY = cirq.GateFamily(cirq.SQRT_ISWAP)
-SQRT_ISWAP_INV_GATE_FAMILY = cirq.GateFamily(cirq.SQRT_ISWAP_INV)
-CZ_GATE_FAMILY = cirq.GateFamily(cirq.CZ)
-PHASED_XZ_GATE_FAMILY = cirq.GateFamily(cirq.PhasedXZGate)
-VIRTUAL_ZPOW_GATE_FAMILY = cirq.GateFamily(cirq.ZPowGate, tags_to_ignore=[ops.PhysicalZTag()])
-PHYSICAL_ZPOW_GATE_FAMILY = cirq.GateFamily(cirq.ZPowGate, tags_to_accept=[ops.PhysicalZTag()])
-COUPLER_PULSE_GATE_FAMILY = cirq.GateFamily(experimental_ops.CouplerPulse)
-MEASUREMENT_GATE_FAMILY = cirq.GateFamily(cirq.MeasurementGate)
-WAIT_GATE_FAMILY = cirq.GateFamily(cirq.WaitGate)
+_SYC_GATE_FAMILY = cirq.GateFamily(ops.SYC)
```

Comment:
I'm leaning towards no because they can be reused for different purposes in the future. Different gate families here are actually used for different things (serializing to DeviceSpecification, specifying target gates of a CompilationTargetGateset, or deserializing to a gateset).

I'll move FSIM gate families into a separate section though so they can be distinguished more easily.

Reply:
Synced offline. Added comments and moved Z gate families to constants.